ocs_ci/utility/ibm_hci.py

```
@@ -62,6 +78,276 @@ def _load_rack_details(self):
+
+        if not candidates:
+            log.warning(
+                "No backup files found to recover rackIP from. "
+                "Rack IP backfill skipped."
+            )
+            return
+
+        patched = False
+        for backup_path in candidates:
+            try:
+                with open(backup_path, "r") as f:
```

Ruff (0.15.20)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)